src/status_im2/contexts/activity_center/style.cljs

```diff
@@ -16,8 +16,7 @@
 
 (defn notification-container
   [index]
-  {:margin-top         (if (zero? index) 0 4)
-   :padding-horizontal 8})
+  {:margin-top         (if (zero? index) 0 4)})
```

Comment:
Indentation flows) You will need to run a `make lint-fix` to fix small stuff like this. Otherwise, the `tests` check won't pass.

Reply:
It's the same Github incident. It's already fixed on my branch, but not shown in the PR.

Reply:
The check itself:
![Screenshot 2023-06-07 at 18 26 05](https://github.com/status-im/status-mobile/assets/5786310/0f8caa6f-0640-43d6-aa93-fdd84b469a82)

Details:
![Screenshot 2023-06-07 at 18 25 50](https://github.com/status-im/status-mobile/assets/5786310/f7d4846b-2aba-4781-a482-c8e7f6eb016c)

Reply:
> It's the same Github incident. It's already fixed on my branch, but not shown in the PR.

Then all perfect! 🤌